pkg/util/randutil/randutil.go

```diff
@@ -0,0 +1,55 @@
+package randutil
```

Comment:
why is this added

Reply:
Only `math/rand` have func `Intn`, need to implement this common func yourself

Reply:
https://pkg.go.dev/math/rand